test/datasets/libritts_test.py

```diff
@@ -33,8 +33,8 @@ def setUpClass(cls):
             os.makedirs(file_dir, exist_ok=True)
             path = os.path.join(file_dir, filename)
 
-            data = get_whitenoise(sample_rate=8000, duration=6, n_channels=1, dtype='int16', seed=i)
-            save_wav(path, data, 8000)
+            data = get_whitenoise(sample_rate=24000, duration=2, n_channels=1, dtype='int16', seed=i)
+            save_wav(path, data, 24000)
```

Comment:
@mthrok, @jimchen90 -- since we now use all synthetic data and dataset, we didn't catch this error. how do you suggest preventing this in the future?

Reply:
There is nothing to prevent here. Since the dataset implementation allows to load arbitral file in the given directory, there is no guarantee that all the data users added have some fixed properties.

Setting this to `24000` here is rather cosmetic, although if it's not the value presented in the paper, it feels wrong.